src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -1016,6 +1018,7 @@ public static final void resetAfterLimitmode() {
     KoLCharacter.resetSkills();
     RequestThread.postRequest(new CharSheetRequest());
     InventoryManager.checkPowerfulGlove();
+    InventoryManager.checkDesignerSweatpants();
```

Comment:
Is this pair of lines worth spinning out into its own method? I presume you've looked through the codebase for wherever powerful glove was called and made sure to call this after -- having a single location would make the next equipment piece like this need adding in only one place.

Reply:
I considered this but I don't think it's worth doing until we get another item of this type - I don't want to abstract too early.

Reply:
Basically I think we could have a data file that says what equipment provides which skills, and do this all automatically based on whether the skill is a combat skill or not. But I don't want to do that right now.